The Newsletter Widget (#2392)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

Below you can see how a ***Contact Form*** will look per default:

![movePage.jpg](images/Contact-form-example.png)

### The Newsletter Widget

The Newsletter Widget, lets you create a sign-up form for your newsletter. This will require some setting up. 

> They way this widget works is that you set it up with your Mailchimp or Campaign monitor, then you add your API key, finally you select the ID for the group this newsletter needs to be sent to.

Reviewer: ```suggestion
The way this widget works is that you set it up with your Mailchimp or Campaign monitor, then you add your API key and set the ID for the list that user will be subscriped to.
```

> You can set-up all of these requred settings in [general-settings](../Settings/General-Settings/index.md#newsletter).

Reviewer: ```suggestion
You can setup all this in the [General Settings](../Settings/General-Settings/index.md#newsletter) page.
```
